src/index.ts

```diff
@@ -300,7 +301,27 @@ export function main(argString = ""): void {
       trackVoteMonster: "free",
       choiceAdventureScript: "",
     });
-    propertyManager.setChoices({ 1341: 1 }); // Cure her poison
+    const average = (items: Item[]) => {
+      return items.reduce((sum, item): number => {
+        return sum + mallPrice(item) / items.length;
+      }, 0);
+    };
+    const halloweinerOptions: { price: number; choiceId: number }[] = (
+      [
+        [$items`bowl of eyeballs, bowl of mummy guts, bowl of maggots`, 1],
+        [$items`blood and blood, Jack-O-Lantern beer, zombie`, 2],
+        [$items`wind-up spider, plastic nightmare troll, Telltale™ rubber heart`, 3],
+      ] as [Item[], number][]
+    ).map(([halloweinerOption, choiceId]) => {
+      return { price: average(halloweinerOption), choiceId: choiceId };
```

Comment:
you can use ```getSaleValue(...$items`bowl of eyeballs, bowl of mummy guts, bowl of maggots`)``` etc to get the average value instead of writing another function